files/default/sql/ddl/3.0.0__initial_tables.sql

```diff
@@ -1373,9 +1373,9 @@ CREATE TABLE `training_dataset_join_condition` (
 CREATE TABLE `on_demand_feature` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
   `on_demand_feature_group_id` int(11) NULL,
-  `name` varchar(1000) COLLATE latin1_general_cs NOT NULL,
+  `name` varchar(63) COLLATE latin1_general_cs NOT NULL,
```

Comment:
The thing is that then it might chop off some existing names etc. in the case of upgrade.

This way I would have said only new clusters will have that new length.